solution/ui/prototype/src/views/ResourcePage.vue

```diff
@@ -58,13 +62,12 @@ export default {
         singleSupList: [],
         sortedSupList: [],
         preSelectedSections:[],
-        preSelectedParts: []
-
-
+        preSelectedParts: [],
+        searchQuery: "",
+        usingSearch: false,
```

Comment:
I don't like extra fields like this. if(searchQuery) is enough to know if search is being used and should always be the same as usingSearch without having to juggle another variable..